### DISCUSSION TITLE
追忆4件套效果有没有触发，胡桃开E和魈开大招不会掉血

### DISCUSSION THREAD

**Discussion by devhxizm** [Melledy/Grasscutter#1440] [Q&A]
昨晚测试发现的问题，追忆4件套的效果打出来的伤害跟预期的伤害有差别，还有胡桃开E不会扣血，魈的问题是我一天群友告知的，我寻思来问一下

**Comment by Liujunhua007** [comment]
与HP有关的东西没有实现，胡桃的E应该是实现了的，关掉无敌更新最新版试试

**Reply by devhxizm** [reply]
最新版是1.2.0的版本吗

**Reply by Liujunhua007** [reply]
不是，是action里最新那个，1.2.2

**Reply by devhxizm** [reply]
那个可以具体问下应该怎么操作吗，我在actions里面最新一栏点开，里面只有一个jar后缀的文件

**Reply by Liujunhua007** [reply]
对，下载那个jar，然后去覆盖目录里的同名jar

**Reply by devhxizm** [reply]
好的，我现在去试试

**Reply by devhxizm** [reply]
> 对，下载那个jar，然后去覆盖目录里的同名jar
我覆盖之后服务器打不开了。。。

**Reply by Liujunhua007** [reply]
打不开？你原来的那个jar是什么名字就保持现在的jar是什么名字，名字变了，cmd里没改，会因找不到闪退。我的是叫这名，每次我都改了才放进去
![image](https://user-images.githubusercontent.com/62129047/177339074-80068f0a-65a5-431f-a65e-0bd1d191a45e.png)

**Reply by devhxizm** [reply]
我改完名之后能进了但是我用原先创建的账户没了

**Reply by devhxizm** [reply]
> 我能进了我之后用完完名的改账户没了

账户问题已经解决了谢谢大佬讲解
